### ISSUE TITLE
linux-4.19.y: ia64 build broken with 4.19.318-rc1

### ISSUE THREAD

**Issue by johnny-mnemonic** [linux-ia64/linux-stable-rc#2]
https://github.com/linux-ia64/linux-stable-rc/actions/runs/9947642032#summary-27481009307

```
[...]
drivers/char/hpet.c: In function 'hpet_read':
drivers/char/hpet.c:311:36: error: 'compat_ulong_t' undeclared (first use in this function)
  311 |                 if (count < sizeof(compat_ulong_t))
      |                                    ^~~~~~~~~~~~~~
drivers/char/hpet.c:311:36: note: each undeclared identifier is reported only once for each function it appears in
[...] 
```

**Comment by johnny-mnemonic** [comment]
When [4e60131](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=4e60131d0d36af65ab9c9144f4f163fe97ae36e8) was included into linux-4.19.y those `compat_` types got introduced. But w/o also including [fb37397](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=fb3739759474d150a9927b920a80ea2afb4c2a51) (which is [in mainline not before v4.20-rc1](https://github.com/torvalds/linux/commit/fb3739759474d150a9927b920a80ea2afb4c2a51)), these types are not known to ia64 (and possibly other architectures).

**Comment by johnny-mnemonic** [comment]
Including [fb37397](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=fb3739759474d150a9927b920a80ea2afb4c2a51) for the GHA build with https://github.com/linux-ia64/linux-stable-rc/commit/d7312a8576493bd97bbce82d2b8425418415023c indeed fixes the issue, see https://github.com/linux-ia64/linux-stable-rc/actions/runs/9970651775#summary-27550542084.

**Comment by johnny-mnemonic** [comment]
In the meantime this was also included in linux-4.19.y, so the extra patch can be dropped and this issue is fixed.
